### ISSUE TITLE
Can't dezoomify https://stadtplan.bvg.de  [new site support]

### ISSUE THREAD

**Issue by ackern** [lovasoa/dezoomify#714]
### Site name and desciption
Berlin bus and subway map

https://stadtplan.bvg.de 

### Example URLs
 
Example of a tile:  https://stadtplan.bvg.de/api/data/10/16/35198/21493.png

 ### Current error message
"Oops, we have a problem. An error occured while trying to dezoomify your image. This is probably because dezoomify did not manage to find a zoomable image at the address you gave."

Chrome extension also doesn't work. The Dezoomify web app worked for this site as it existed in 2019 but (1) the site has since changed (2) I'm not completely sure how I got it to work. I have notes saying the following string worked back then. I guess I pasted this into the web app, does that sound right?

/dezoom.sh -x 17593 -X 17609 -Y 22014 -y22026 https://fahrinfo.bvg.de/tiles/base/15/%X/%Y.png

(with variations in the x and y numbers depending on how much of the map I wanted. Note that back then the main URL was fahrinfo.bvg.de and now it's stadtplanbvg.de )

(3) I don't know how to get the coordinates of the tiles for the corners (x and y values). I know how to get URLs for tiles in Network Inspector but I don't know how to choose the corners because "Inspect Element" doesn't let you choose which tiles to inspect. Somehow I got them in 2019 though.

**Comment by ackern** [comment]
I figured out the URLs of the tiles by brute force but I don't know if that helps or what to do with them - 

top left corner:		https://stadtplan.bvg.de/api/data/20/14/8790/5361.png
top right corner:	https://stadtplan.bvg.de/api/data/20/14/8822/5361.png
bottom left corner:	https://stadtplan.bvg.de/api/data/20/14/8790/5391.png
bottom right corner:	https://stadtplan.bvg.de/api/data/20/14/8822/5391.png

They could be off by one or two e.g. conceivably the columns could go from 8789 to 8823 instead of 8790 to 8222.

Also I tried these in the web app and dezoomify-rs; didn't work - 

https://stadtplan.bvg.de/api/data/20/14/-/-.png
https://stadtplan.bvg.de/api/data/20/14/{{X}}/{{Y}}.png

**Comment by DigitalCurator** [comment]
From my knowledge Dezoomify cannot download street map directory like the website you stated. You need mapping software for that purpose,

**Comment by ackern** [comment]
> From my knowledge Dezoomify cannot download street map directory like the website you stated.

Yes it most certainly can! Did you read my post all the way through? I said Dezoomify used to work perfectly for the site in question and I posted the code that used to work. I have the map that Dezoomify downloaded around 2019 right here on my computer.  I'm looking at it right now. You will have to be more clear about what exactly is a "street map directory" because it sounds like something competely unrelated to this site. The "map" is just an image, that's all. No different from a regular photograph or whatever. There is no data about streets whatsoever, no text or numeric data, nothing. Just pure png images.

**Comment by DigitalCurator** [comment]
**Not** dezoomify
![BE85B413-D9B9-4006-9C77-EC2BB3D16900](https://github.com/lovasoa/dezoomify/assets/133426195/edcc4510-6760-48fa-a1dc-a6a749c32111)

**Comment by lovasoa** [comment]
If the URL scheme has not changed, you can download it with dezoomify-rs : https://github.com/lovasoa/dezoomify-rs/wiki/Usage-example-for-the-custom-YAML-dezoomer
